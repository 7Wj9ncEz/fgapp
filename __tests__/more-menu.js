import 'react-native';
import React from 'react';
import MoreMenu from '../components/more-menu.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <MoreMenu/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

