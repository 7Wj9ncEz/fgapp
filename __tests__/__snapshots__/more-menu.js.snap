exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "lightblue",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    More opstions
  </Text>
</View>
`;
